src/index.ts

```diff
@@ -75,6 +76,16 @@ export const LD_CLIENT = Symbol() as InjectionKey<LDClient>
  */
 export const LD_FLAG = Symbol() as InjectionKey<<T>(flagKey: string, defaultFlagValue?: T | undefined) => FlagRef<T>>
 
+/**
+ * Will hold the LaunchDarkly instance.
+ */
+export const launchDarklyClient = ref() as Ref<LDClient>
```

Comment:
I think this breaks scoping/encapsulation. As this ref would be the same even with two apps both using the LDPlugin. 

Where the provide/inject for the built-in ready and client theoretically would be scoped.

I think that route guards support inject as of vue 3.3. https://router.vuejs.org/guide/advanced/navigation-guards.html#Global-injections-within-guards

I need to look into what the implications of us upgrading would be. 

I think, in that case, useLDClient and useLDReady could be used without duplicating these refs and without causing a scope problem.